python/mxnet/model.py

```diff
@@ -431,6 +431,7 @@ def load_params(prefix, epoch):
     aux_params = {}
     if not save_dict:
         logging.warning("Params file '%s' is empty", '%s-%04d.params' % (prefix, epoch))
+        return (arg_params, aux_params)
```

Comment:
Thanks for your contribution!
In the original code, the function will return empty weights in the line 440 when loading an empty file.